subcommittees/mentoring/mission-objectives.md

```diff
@@ -0,0 +1,48 @@
+# Mission
+
+The Mentoring Subcommittee was established in 2015 to develop and maintain a mentorship program to support instructors as they progress through training, teaching, curriculum development, and other community-related activities. We help promote community-building and networking by providing (virtual) spaces where instructors from all over the world can share teaching success stories and discuss strategies for overcoming challenges. 
+
+# Objectives
+
+* Need to add
+
+# Activities
+
+## Current
+
+(See the [README](README.md) for how to participate in these activities.)
+
+* Instructor Discussion Sessions: one hour weekly discussions for instructors at 
+past and upcoming workshops
+	> Satisfies objectives x, y, z
+* Mentoring Program: small groups of newer instructors/community members being mentored 
+by 1-2 experienced instructors/community members
+	> Satisfies objectives x, y, z
+* Regional Communities Sub-Committee: Sub-group of local community leaders to share knowledge, tips and tricks for organizing local SWC/DC groups
+	> Satisfies objectives x, y, z
+
+## Wishlist
+
```

Comment:
This is a wishlist of potential future efforts.  Is this the place to store this information?  Is there anything missing?